src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -619,8 +623,10 @@ static void update_lb_connectivity_status_locked(
 
   if (GRPC_TRACER_ON(grpc_lb_glb_trace)) {
     gpr_log(
-        GPR_INFO, "Setting grpclb's state to %s from new RR policy %p state.",
-        grpc_connectivity_state_name(rr_state), (void*)glb_policy->rr_policy);
+        GPR_INFO,
+        "[GLB %p] Setting grpclb's state to %s from new RR policy %p state.",
```

Comment:
Instead of "GLB", let's use "grpclb" in the log messages.  I think it will be easier to read for the person reading the logs.